spec/requests/account/users_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Account::UsersController, type: :request do
+  let(:admin) { FactoryBot.create(:user, :admin) }
```

Comment:
У вас здається заінклуджені хелпери факторі бота, тому можна просто так
```suggestion
  let(:admin) { create(:user, :admin) }
```